flake.nix

```diff
@@ -108,90 +121,101 @@
+                # Add `try-import %workspace%/.bazelrc.local` to your .bazelrc to include these
+                # flags when running Bazel in a nix environment. These are the libs and frameworks
+                # used by darwin.
 
-                  build --@rules_rust//:extra_exec_rustc_flags=-L${pkgs.libiconv}/lib
-                ''
-              else "";
-          in
-          pkgs.mkShell rec {
-            buildInputs = with pkgs; [
+                build --@rules_rust//:extra_exec_rustc_flags=-L${pkgs.libiconv}/lib
+              ''
+            else
+              "";
+        in
+        pkgs.mkShell rec {
+          buildInputs =
+            with pkgs;
+            [
               # === Bazel ===
               bazel
               # === Graal dependencies ===
               libxcrypt-legacy
               # === Rust dependencies ===
               openssl.dev
               pkg-config
-            ] ++ (if !isOnLinux then [
-              # === macOS-specific dependencies ===
-              libiconv # Required by `sysinfo` (via `ide_ci`).
-            ] else [ ]);
+            ]
+            ++ (
+              if !isOnLinux then
+                [
+                  # === macOS-specific dependencies ===
+                  libiconv # Required by `sysinfo` (via `ide_ci`).
+                ]
+              else
+                [ ]
+            );
 
-            packages = with pkgs; [
-              # === Shims (highest precedence) ===
-              pnpm-shim
-              rustup-shim
-              # === TypeScript dependencies ===
-              nodejs_22
-              corepack
-              # === Electron ===
-              electron
-              # === node-gyp dependencies ===
-              python3
-              gnumake
-              # === WASM parser dependencies ===
-              rust
-            ];
+          packages = with pkgs; [
+            # === Shims (highest precedence) ===
+            pnpm-shim
+            rustup-shim
+            # === TypeScript dependencies ===
+            nodejs_22
+            corepack
+            # === Electron ===
+            electron
+            # === node-gyp dependencies ===
+            python3
+            gnumake
+            # === WASM parser dependencies ===
+            rust
+          ];
 
-            shellHook = ''
-              export LD_LIBRARY_PATH="${pkgs.lib.makeLibraryPath buildInputs}:$LD_LIBRARY_PATH"
+          shellHook = ''
+            export LD_LIBRARY_PATH="${pkgs.lib.makeLibraryPath buildInputs}:$LD_LIBRARY_PATH"
 
-              if ! readlink "''${PWD}/.bazelrc.local" >/dev/null \
-                || [[ $(readlink "''${PWD}/.bazelrc.local") != ${bazelrc} ]]; then
-                echo 1>&2 "Darwin: updating $PWD repository"
-                [ -L .bazelrc.local ] && unlink .bazelrc.local
+            if ! readlink "''${PWD}/.bazelrc.local" >/dev/null \
+              || [[ $(readlink "''${PWD}/.bazelrc.local") != "${bazelrc}" ]]; then
```

Comment:
@indiv0 lmk whether this is a correct fix btw

(also IDK what's up with the formatting changes, maybe i'm using the wrong formatter in VSCode?)